hypothesis-python/tests/numpy/test_gen_data.py

```diff
@@ -530,7 +521,7 @@ def test_broadcastable_shape_bounds_are_satisfied(shape, data):
             label="bshape",
         )
     except InvalidArgument:
-        return
+        assume(False)
```

Comment:
I mean for this test to throw general input values, invalid and valid alike, at `broadcastable_shapes` to see if we can get it to violate the bounds that we promise to the user or get it to hit an assertion error. For this reason, I specified `return` here instead of `assume(False)` , since I'm not really meaning to avoid invalid argument configurations. 

I'm fine with this change, but it makes me nervous that this may fail a health check for filtering out examples.

Reply:
I do see what you mean - I added this to ensure that we actually do run the block below, as with `return` it's possible for this test to pass without ever generating a valid input.

For the other, all I can do is observe that more than half of all inputs are valid, and shrinking earlier values or all simultaneously will remain valid. More importantly though, it's passing in practice!